src/aave-v3/SupplyHarvestVault.sol

```diff
@@ -56,6 +56,11 @@ contract SupplyHarvestVault is ISupplyHarvestVault, SupplyVaultBase, OwnableUpgr
     uint16 public harvestingFee; // The fee taken by the claimer when harvesting the vault (in bps).
     ISwapper public swapper; // Swapper contract to swap reward tokens for underlying asset.
 
+    /// @dev This empty reserved space is put in place to allow future versions to add new
+    /// variables without shifting down storage in the inheritance chain.
+    /// See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
+    uint256[48] private __gap;
```

Comment:
What is the purpose of this gap? `SupplyHarvestVault` is not supposed to be extended

I understand we could do this in each of our smart contracts... But it's not what we've done until now (and a lot of contracts are deployed)

Reply:
We should have done it IMO as now we'll face some issues with those vaults if we want to upgrade them and add storage variables. 

> SupplyHarvestVault is not supposed to be extended

At least for now, I cannot say for sure in the future, so I think there's no cost to add this rn. What do you think?

Reply:
Mh ok but then we could add this gap sometime later when we do extend the implementation (this would be lean)
It's not a veto from me, it's just not lean

We still have an issue with the current deployed SupplyVaultBase though

Reply:
Fixed in #116